lib/formatters/tap.js

```diff
@@ -66,6 +66,7 @@ module.exports = function(results) {
                 // The first error will be logged as message key
                 // This is to adhere to TAP 13 loosely defined specification of having a message key
                 if ("message" in diagnostics) {
+                    /* istanbul ignore if this is never going to happen we are not setting messages */
```

Comment:
Or this one. We are creating `diagnostics` object just few lines above, and we are not adding `messages` property to it, so this if statement is impossible to hit.